### ISSUE TITLE
skid

### ISSUE THREAD

**Issue by 15chrhay** [xgladius/opcodeIntegrity#1]
ok

**Comment by rakion99** [comment]
and here we go with the retards with their useless hate

**Comment by 15chrhay** [comment]
> and here we go with the retards with their useless hate

No, I just don't like skids, therefore, I don't like you or xgladius. Simple? Or is that too complicated for you?

**Comment by xgladius** [comment]
excuse me who even are you?

**Comment by kill081377** [comment]
lol @15chrhay just go die with ur gay ass name deadpool lol your life is a skid leave the poor man alone

**Comment by 15chrhay** [comment]
> lol @15chrhay just go die with ur gay ass name deadpool lol your life is a skid leave the poor man alone

Nice meme but a skid isn't allowed to use insults on their superiors. I suggest for you to not do it again :)

**Comment by 15chrhay** [comment]
2 weeks later and you're still a skid
fix it.

**Comment by kill081377** [comment]
> 2 weeks later and you're still a skid
> fix it.

Stfu you're a skid dude can you just get a life and stop getting sucked byyour mom?, i suggest for you to stop or you'll get fucked in the ass little nigga
